### PR TITLE
React: Select user country by IP address, add missing countries

### PR DESCRIPTION
**Pull request by msieczko**
Rename code -> currency

Introduce Country type, move country data to a separate file

Add remaining EU countries

Add node-fetch dependency to react project

Add nock and chai-as-promised dev dependencies to react project

Create service for fetching geolocation data of IP address

Add @restless/sanitizers dependency

Sanitize API response

Add remaining flag images and country codes

Add white background to flag images

Replace old smaller flags with new ones

Refactor react project config

Add IPGeolocationApi configs

Resolve problem with error in nock dependency type definitions

Resolve problem with request header

Select user country automatically

### CODE REVIEW

File: universal-login-react/package.json [hunk 1]
```diff
@@ -16,11 +16,13 @@
   },
   "dependencies": {
     "@ramp-network/ramp-instant-sdk": "^1.0.1",
+    "@restless/sanitizers": "^0.2.2",
     "@universal-login/commons": "0.3.40",
     "@universal-login/sdk": "0.3.40",
     "dotenv": "^8.1.0",
     "ethers": "^4.0.36",
     "lodash.range": "^3.2.0",
+    "node-fetch": "^2.6.0",
```
Comment (marekkirejczyk) [inline universal-login-react/package.json]: Do we really want node-fetch as a dependency in react project?

Comment (msieczko) [reply universal-login-react/package.json]: It's necessary only for test purposes. Moved to dev dependencies.

File: universal-login-react/test/integration/http/IPGeolocationService.ts [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+
+      await expect(service.getCountryCode()).to.eventually.eq('PL');
+    });
+
+    it('throws error in case of invalid response', async function () {
+      nock(baseUrl)
+        .get('/check?access_key=ABCD&fields=country_code')
+        .reply(200, {country_code: null});
+
+      await expect(service.getCountryCode()).to.be.rejected;
```
Comment (marekkirejczyk) [inline universal-login-react/test/integration/http/IPGeolocationService.ts]: `rejectedWith`?

File: universal-login-react/src/config/index.ts [hunk 3]
```diff
@@ -1,11 +1,17 @@
+import {ReactConfig} from './types';
+
 require('dotenv').config();
 import {ETHER_NATIVE_TOKEN} from '@universal-login/commons';
 
-const developmentConfig = {
+const developmentConfig: ReactConfig = {
```
Comment (marekkirejczyk) [inline universal-login-react/src/config/index.ts]: I am not sure if `ReactConfig` is proper names. Sounds like general React config. Not like Universal Login React Components config. Perhaps `ULReactConfig`?

Comment (msieczko) [reply universal-login-react/src/config/index.ts]: Took inspiration from `SdkConfig`. I'll rename `ReactConfig` -> `Config` for now, the config will surely be revisited in the future.

File: universal-login-react/test/integration/http/IPGeolocationService.ts (hunk 2 above)
Comment (msieczko) [reply universal-login-react/test/integration/http/IPGeolocationService.ts]: I was working on a fix, will include it in the next PR.

**Pull request merged**